src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -0,0 +1,224 @@
+            ),
+        ),
+        (
+            np.random.randint(255, size=(218, 220, 3), dtype=np.uint8),
+            transforms.Compose(
+                [
+                    transforms.Pad((2, 3), padding_mode="edge"),
+                    transforms.ToTensor(),
+                ]
+            ),
+        ),
+        (
+            np.random.randint(255, size=(218, 220, 3), dtype=np.uint8),
+            transforms.Compose(
+                [
+                    transforms.Pad((2, 3), padding_mode="reflect"),
+                    transforms.ToTensor(),
+                ]
+            ),
+        ),
+        (
+            np.random.randint(255, size=(218, 220, 3), dtype=np.uint8),
+            transforms.Compose(
+                [
+                    transforms.Pad((2, 3), padding_mode="symmetric"),
+                    transforms.ToTensor(),
+                ]
+            ),
+        ),
+    ],
+)
+def test_Pad(test_input, preprocess_pipeline):
+    torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline)
+    assert np.max(np.absolute(torch_result - ov_result)) < 4e-05
+
+
+def test_CenterCrop():
+    test_input = np.random.randint(255, size=(260, 260, 3), dtype=np.uint16)
+    preprocess_pipeline = transforms.Compose(
+        [
+            transforms.CenterCrop((224)),
+            transforms.ToTensor(),
+        ]
+    )
+    torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline)
+    assert np.max(np.absolute(torch_result - ov_result)) < 4e-05
+
+
+def test_Grayscale():
+    test_input = np.random.randint(255, size=(224, 224, 3), dtype=np.uint16)
+    preprocess_pipeline = transforms.Compose([transforms.ToTensor(), transforms.Grayscale()])
+    torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline, input_channels=1)
+    assert np.max(np.absolute(torch_result - ov_result)) < 1e-04
+
+
+def test_Grayscale_num_output_channels():
+    test_input = np.random.randint(255, size=(224, 224, 3), dtype=np.uint16)
+    preprocess_pipeline = transforms.Compose([transforms.ToTensor(), transforms.Grayscale(3)])
+    torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline)
+    assert np.max(np.absolute(torch_result - ov_result)) < 1e-04
+
+
+def test_pipeline_1():
+    test_input = np.random.randint(255, size=(260, 260, 3), dtype=np.uint16)
+    preprocess_pipeline = transforms.Compose(
+        [
+            transforms.Resize(256, interpolation=transforms.InterpolationMode.NEAREST),
+            transforms.CenterCrop((216, 218)),
+            transforms.Pad((2, 3, 4, 5), fill=3),
+            transforms.ToTensor(),
+            transforms.ConvertImageDtype(torch.float32),
+            transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+        ]
+    )
+    torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline)
+    assert np.max(np.absolute(torch_result - ov_result)) < 4e-05
+
+
+def test_pipeline_2():
```

Comment:
I took a look at the Pytorch tutorials, and it seems like we covered basic tutorial from here: https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html in tests. But can we also cover this one: https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html#putting-everything-together

Reply:
Done in `test_pipeline_3()`.